src/test/java/util/loader/FileLoaderTest.java

```diff
@@ -0,0 +1,25 @@
+package util.loader;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static util.loader.FileLoader.loadCSV;
+
+import java.util.Scanner;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class FileLoaderTest {
+
+    @Test
+    @DisplayName("리소스에서 파일을 읽어올 수 있다.")
+    void successLoadFile() {
+        Scanner scanner = loadCSV("src/test/resources/attendances.csv");
```

Comment:
```suggestion
        Scanner scanner = FileLoader.loadCSV("src/test/resources/attendances.csv");
```

`assertThat`처럼 static import가 가독성을 높여주는 경우도 있지만 일반적인 상황에서는 static import 사용은 피하는 게 좋아요. 어떤 클래스에서 온 메서드인지 파악하기 어렵습니다!